source/MRViewer/MRPalette.h

```diff
@@ -23,16 +23,7 @@ class Palette
 {
 public:
     // preset palette colors
-    static inline const std::vector<Color> DefaultColors =
-    {
-        Color( Vector4f { 0.1f, 0.25f, 1.0f, 1.f } ),// almost blue   |
-        Color( Vector4f { 0.15f, 0.5f, 0.75f,1.f } ),//               | -> "inside" the ref mesh
-        Color( Vector4f { 0.2f, 0.75f, 0.5f, 1.f } ),//               |
-        Color( Vector4f { 0.25f, 1.0f, 0.25f,1.f } ),// almost green -> zero distance
-        Color( Vector4f { 0.5f, 0.75f, 0.2f, 1.f } ),//               |
-        Color( Vector4f { 0.75f, 0.5f, 0.15f,1.f } ),//               | -> "outside"
-        Color( Vector4f { 1.0f, 0.25f, 0.1f, 1.f } ),// almost red    |
-    };
+    MRVIEWER_API static const std::vector<Color> DefaultColors;
```

Comment:
better create static function that will return it (mb `static` -> `extern`)

Reply:
Yes, it is better, but it will break the current interface. Let us keep it as is for now.